docs/collecting-data/code-generation/client-side-validation/index.md

```diff
@@ -57,13 +57,59 @@ As we can observe, the value passed as the `id` attribute is violating the schem
 
 Currently the validation information will include attributes that can help point out the issue happening at the schema level, plus the stack trace revealing the caller of the function.
 
+## Entity cardinality rules validation example
+
+:::info
+This feature is available since version 0.3.1 of Snowtype for the [Browser Tracker](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/quick-start-guide/?platform=browser) in TypeScript.
+:::
+
+Cardinality rules allow you to specify the expected number of an entity taking part in an Event Specification. You would use this capability to ensure the correct number of entities are getting sent alongside your event. E.g. 
+- `Exactly 1` 
+- `At least 1`
+- `Between 1 and 2`
+
+By using Snowtype client-side validations you will get notified right in your browser that there is a violation of cardinality rules for an Event Specification.
```

Comment:
I would mention why it is cool, something like:
"this is very important when developing or qa"